test/visual/test.js

```diff
@@ -83,4 +83,10 @@ gemini.suite('vaadin-custom-field', function(rootSuite) {
       .capture('default');
   });
 
+  gemini.suite('lumo-tests-whitespace', function(suite) {
+    suite
+      .setUrl('lumo.html?theme=lumo')
+      .setCaptureElements('#lumo-test-whitespace')
+      .capture('default');
```

Comment:
As discussed, we need to figure out why there is such a big difference in the offset between Chrome and Firefox